src/Nethermind/Nethermind.JsonRpc/Modules/Admin/AdminRpcModule.cs

```diff
@@ -193,4 +200,35 @@ public ResultWrapper<string> admin_verifyTrie(BlockParameter block)
 
         return ResultWrapper<string>.Success("Starting.");
     }
+
+    public ResultWrapper<string> admin_subscribe(string subscriptionName, string? args = null)
+    {
+        try
+        {
+            ResultWrapper<string> successfulResult = ResultWrapper<string>.Success(_subscriptionManager.AddSubscription(Context.DuplexClient, subscriptionName, args));
+            return successfulResult;
+        }
+        catch (KeyNotFoundException)
```

Comment:
To be honest we could rework internals to not throw this exception as we can expect it.

`bool TryAddSubscription(Context.DuplexClient, subscriptionName, args, out Subscription?));`